media/xhtml1-transitional.dtd

```diff
@@ -0,0 +1,1201 @@
+<!--
```

Comment:
I don't think including this file is necessary - an alternative could be changing doctype in `base.html` to: `<!doctype html>`